parm/atm/obs/testing/ompsnp_npp.yaml

```diff
@@ -0,0 +1,274 @@
+obs space:
+  name: ompsnp_npp
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: !ENV ompsnp_npp_obs_${CDATE}.nc4
+    obsgrouping:
+      group variables: ["latitude"]
+      sort variable: "pressure"
+      sort order: "ascending"
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: !ENV ompsnp_npp_diag_${CDATE}.nc4
+  io pool:
+    max pool size: 1
+  simulated variables: [ozoneTotal]
```

Comment:
is it ozoneTotal? isn't there an ozoneLayer?

Reply:
I think this dataset is a 22 layer profile right? or am I misremembering

Reply:
I checked GSI fix file [`global_ozinfo.txt`](https://github.com/NOAA-EMC/GSI-fix/blob/develop/global_ozinfo.txt).   `ompsnp_npp` is 22 layers.  `ompstc8_npp` is total column.   Do we treat these observations the same way in JEDI?

Reply:
However, `ncdump -hcs gdas.t18z.ompsnp_npp.2021122318.nc4` shows that the file contains `ozoneTotal`.   I'm confused.   Shouldn't this file contain layer ozone?

Reply:
A check of `diag_ompsnp_npp_ges.2023030900_ges.nc4` shows `Reference_Pressure(nobs)` and `Observation(nobs)` are in the file.   `ncdump` of the reference pressure shows a repeated pattern of 22 values
```
 Reference_Pressure = 0.101325, 0.1600935, 0.2543257, 0.4032735, 0.6393607,
    1.01325, 1.600935, 2.543257, 4.032735, 6.393608, 10.1325, 16.00935,
    25.43258, 40.32735, 63.93607, 101.325, 160.0935, 254.3257, 403.2735,
    639.3608, 1013.25, 0, 0.101325, 0.1600935, 0.2543257, 0.4032735,
    0.6393607, 1.01325, 1.600935, 2.543257, 4.032735, 6.393608, 10.1325,
    16.00935, 25.43258, 40.32735, 63.93607, 101.325, 160.0935, 254.3257,
    403.2735, 639.3608, 1013.25, 0, 
```
A check of `Observation` shows profiles as a sequence of 22 obs
```
 Observation = 0.02, 0.04, 0.11, 0.26, 0.59, 1.21, 2.39, 4.08, 6.34, 9.95,
    15.61, 23.42, 34.94, 53, 70.56, 76.32, 71.79, 42.93, 16.43, 10.84, 12.53,
    453.36, 0.02, 0.04, 0.1, 0.25, 0.6, 1.29, 2.45, 4.18, 6.63, 10.47, 16.57,
    24.84, 35.4, 50.41, 66.37, 74.02, 71.67, 43.47, 16.75, 11.09, 12.84,
    449.46,
```
The 22nd obs appears to be the column total.

A check of `Longitude` shows repetitions of 22.
```
 Longitude = 120.68, 120.68, 120.68, 120.68, 120.68, 120.68, 120.68, 120.68,
    120.68, 120.68, 120.68, 120.68, 120.68, 120.68, 120.68, 120.68, 120.68,
    120.68, 120.68, 120.68, 120.68, 120.68, 128.52, 128.52, 128.52, 128.52,
    128.52, 128.52, 128.52, 128.52, 128.52, 128.52, 128.52, 128.52, 128.52,
    128.52, 128.52, 128.52, 128.52, 128.52, 128.52, 128.52, 128.52, 128.52,
```
It seems the GSI encodes omps pressure layer data as individual obs instead of profiles.  Is this correct?

Reply:
yes I believe so. Would need to ask others (@HaixiaLiu-NOAA ?) to confirm. I guess the question is for this PR, do we merge as-is and fix later or fix ompsnp_npp to be `ozoneLayer`? This will require the converted data to be `ozoneLayer` not `ozoneTotal` for the nadir profile observation type.